api-docs/api-reference/methods/users/post-reset-pwd-v2.0.rst

```diff
@@ -0,0 +1,72 @@
+.. _post-reset-pwd-v2.0:
+
+Reset password
+~~~~~~~~~~~~~~~
+
+.. code::
+
+    POST /v2.0/users/RAX-AUTH/reset-pwd
+
+Use the Reset Password operation to update users password.
+
+This table shows the possible response headers for this operation:
+
+.. csv-table::
+   :header: Name, Type, Description
+   :widths: 2, 2, 2
+
+   X-User-Name, Header, Value for name provided on operation
+
```

Comment:
Add a response section and move this table underneath it. Also, add a response code sample.